graylog2-server/src/main/java/org/graylog2/indexer/searches/SearchesClusterConfig.java

```diff
@@ -115,7 +121,8 @@ public static SearchesClusterConfig create(@JsonProperty("query_time_range_limit
                                                @JsonProperty("analysis_disabled_fields") @Nullable Set<String> analysisDisabledFields,
                                                @JsonProperty("auto_refresh_timerange_options") @Nullable Map<Period, String> autoRefreshTimerangeOptions,
                                                @JsonProperty("default_auto_refresh_option") Period defaultAutoRefreshOption,
-                                               @JsonProperty("quick_access_timerange_presets") @Nullable List<TimerangePreset> quickAccessTimerangePresets
+                                               @JsonProperty("quick_access_timerange_presets") @Nullable List<TimerangePreset> quickAccessTimerangePresets,
+                                               @JsonProperty("cancel_after_interval") @Nullable Integer cancelAfterSeconds
```

Comment:
Is `cancel_after_interval` correct here? everywhere else it's `cancel_after_seconds`